src/common/pool_map.c

```diff
@@ -1349,26 +1339,93 @@ uuid_compare_cb(const void *a, const void *b)
 	return uuid_compare(*ua, *ub);
 }
 
+static int
+add_domains_to_pool_buf(struct pool_map *map, struct pool_buf *map_buf,
+			int map_version,
+			int ndomains, const uint32_t *domains)
+{
+	int		i, rc;
+	uint32_t	num_comps;
+	uint8_t		new_status;
+	const int	TUPLE_SIZE = 3;
+	uint32_t	domains_found;
+
+	if (map != NULL) {
+		new_status = PO_COMP_ST_NEW;
+		num_comps = pool_map_find_domain(map, PO_COMP_TP_RACK,
+						 PO_COMP_ID_ALL, NULL);
+	} else {
+		new_status = PO_COMP_ST_UPIN;
+		num_comps = 0;
+	}
+
+	/*
```

Comment:
Minor nit, you need the /** instead of /*

Reply:
Hm, `/**` is the format for a doxygen comment, which can be used to generate external documentation. I thought in these cases, it wasn't appropriate to include these annotations in external docs. Do you disagree?

Reply:
In the section "Comments" of https://wiki.hpdd.intel.com/display/DC/C+Style+Guide it shows the use of the /**.  In the section "Functions & Variables" it shows additional instructions for the DOxygen style comments.  Since the "Comments" section is not part of the "Functions & Variables" section, it seems to stand alone.  I didn't see any section of the style guide that endorses the /* comment style though.

Reply:
Huh, the text in that section doesn't mention the second asterisk, but the examples show it. I know we're not that consistent on this in the code, but I always interpreted it as: use doxygen comments for the public functions/variables/structs, but use ordinary comments for internal annotations of functions. Could use some clarification in the style guide, I think.

Reply:
I am not too concerned with the comment style one way or the other.  It was just something I noticed while looking at your code.  As I said, it's a just a nit.